### PR TITLE
feat: pebble, default deneb epoch 4, fix readme, default count 2

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

**Review by parithosh — APPROVED** [review]

File: src/package_io/parse_input.star [hunk 1]
```diff
@@ -382,15 +382,15 @@ def default_participant():
...
+        "count": 2,
     }
 
 
 def get_default_mev_params():
     return {
         "mev_relay_image": MEV_BOOST_RELAY_DEFAULT_IMAGE,
         # TODO replace with flashbots/builder when they publish an arm64 image as mentioned in flashbots/builder#105
-        "mev_builder_image": "ethpandaops/flashbots-builder:main",
+        "mev_builder_image": "flashbots/builder:1.13.2.4844.dev5-6e6c723",
```
Comment (h4ck3rk3y) [inline src/package_io/parse_input.star:393]: should we use a dev version?

Comment (barnabasbusa) [reply src/package_io/parse_input.star:393]: They don't have a recent enough version which supports pebble. So we kinda need to.

File: network_params.json [hunk 2]
```diff
@@ -22,7 +23,7 @@
 		"preregistered_validator_keys_mnemonic": "giant issue aisle success illegal bike spike question tent bar rely arctic volcano long crawl hungry vocal artwork sniff fantasy very lucky have athlete",
 		"genesis_delay": 120,
 		"capella_fork_epoch": 0,
-		"deneb_fork_epoch": 500,
+		"deneb_fork_epoch": 4,
```
Comment (kamilchodola) [inline network_params.json:26]: Is the MEV builder ready for that?

Comment (barnabasbusa) [reply network_params.json:26]: I'm working on testing this right now.

**Comment by barnabasbusa** [comment]
Closed in favor of #288

**Pull request closed without merging**